packages/backend/src/open_payments/payment/outgoing/service.test.ts

```diff
@@ -696,7 +709,6 @@ describe('OutgoingPaymentService', (): void => {
         const amountSent = receiveAmount.value * BigInt(2)
         await expectOutcome(payment, {
           accountBalance: payment.sendAmount.value - amountSent,
-          amountSent,
```

Comment:
I think we still want to check specific expected `amountSent`s, but now that can happen in the individual tests.